docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -159,18 +159,29 @@ start the {dfeed} for the change to be applied.
 == 7. Set the model memory limit
 
 The `model_memory_limit` job configuration option sets the approximate maximum 
-amount of memory resources required for analytical processing. If this variable 
-is set too low for the job and the limit is approached, data pruning becomes 
-more aggressive. Upon exceeding this limit, new entities are not modeled.
-
-Use model memory estimation to have a better picture of the memory needs of the 
-model. Model memory estimation happens automatically when you create the job in 
-{kib} or you can call the 
-{ref}/ml-estimate-model-memory.html[Estimate {anomaly-jobs} model memory API] 
-manually. The estimation is based on the analysis configuration details for the 
-job and cardinality estimates for the fields it references. You can update the 
-memory settings of an existing job, but the job must be closed.
-
+amount of memory resources required for analytical processing. When you create
+an {anomaly-job} in {kib}, it provides an estimate for this limit. The estimate 
+is based on the analysis configuration details for the job and cardinality 
+estimates for the fields it references. You can also obtain this information
+from the
+{ref}/ml-estimate-model-memory.html[estimate {anomaly-jobs} model memory API].  
+
+If you change the resources available on your {ml} nodes or the characteristics 
+of your data, the memory requirements might also change. You can increase the 
+model memory limit for an existing job while it is closed. To decrease the limit, 
+you must clone and re-run the job.
```

Comment:
You can also decrease the model memory limit while the job is closed, providing you do not reduce it below the size reported in the latest model size stats.  (The ability to reduce model memory limit was added in 6.3.)

Reply:
Thanks!  I've tried to make that clearer in the API too via https://github.com/elastic/elasticsearch/pull/74517